source/common/config/config_provider_impl.cc

```diff
@@ -0,0 +1,125 @@
+#include "common/config/config_provider_impl.h"
+
+namespace Envoy {
+namespace Config {
+
+std::ostream& operator<<(std::ostream& os, ConfigProviderInstanceType type) {
```

Comment:
where is this used?

Reply:
It was needed for an fmt::format() statement, but I have now removed it.